models/users.js

```diff
@@ -49,6 +50,17 @@ const addOrUpdate = async (userData, userId = null) => {
      */
     userData.roles = { archived: false };
     userData.incompleteUserDetails = true;
+    /**
+     * Adding default status to user enable to query for idle user
+     * This can be used in /users/idle
+     * https://github.com/Real-Dev-Squad/website-backend/issues/635
+     * The reason I am checking whether status exists or not before adding
+     * 1. If we are adding the user with some status it will not get override
+     * 2. Some tests are depends on the status of user, We are adding the user before each tests, those will consider as new user only and the status will be override if we don't check
+     */
+    if (userData.status === undefined) {
```

Comment:
Ideally we should not write code to pass the test cases. 
If test cases are breaking we can edit them

Reply:
Thank you for pointing out. Made the change. please check @samarpan1738